### PR TITLE
ODC-7336: automation for customization-of-catalog-add-page-form feature file

### PR DESCRIPTION
**Pull request by The-Anton**
**Description:**

- Automation of customization-of-catalog-addPage-form-view feature file

**Jira Id:**
- Story: 
      -  [[ODC-7336](https://issues.redhat.com/browse/ODC-7336))

**Checks for approving Epic scenarios Automation PR:**
- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**
- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
BRIDGE_KUBEADMIN_PASSWORD=BrCQB-qAJti-7GkZg-GA2bd
export BRIDGE_KUBEADMIN_PASSWORD
BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.ci-ln-c41px9b-72292.origin-ci-int-gce.dev.rhcloud.com/
export BRIDGE_BASE_ADDRESS
oc login https://api.ci-ln-c41px9b-72292.origin-ci-int-gce.dev.rhcloud.com:6443 -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
./test-cypress.sh -p dev-console
```

**Screen shots**:

Screenshot 2023-07-20 at 3 31 27 PM

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/support/step-definitions/customization/customization-of-catalog-addPage-form-view.ts [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+Given('user disables all the items in Add page', () => {
+  cy.byTestID(customizationPO.addPage.formSection)
+    .find(customizationPO.addPage.addAllAction)
+    .scrollIntoView()
+    .should('be.visible')
+    .click();
+});
+
+Given('user disables "Import from Git" the items in Developer Catalog', () => {
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/support/step-definitions/customization/customization-of-catalog-addPage-form-view.ts:111]: Don't use items in "" directly here instead use a variable 
```suggestion
Given('user disables {string} the items in Developer Catalog', (item: string) => {
```

Comment (The-Anton) [reply frontend/packages/dev-console/integration-tests/support/step-definitions/customization/customization-of-catalog-addPage-form-view.ts]: Sure, made all the changes.

**Comment by sanketpathak** [comment]
Tests run fine remotely
Screenshot 2023-07-21 at 1 10 29 AM
/lgtm
/approve

**Review by sanketpathak — APPROVED** [review]

**Comment by sanketpathak** [comment]
Changes only affect e2e tests
/label docs-approved
/label px-approved
/label qe-approved

**Comment by sanketpathak** [comment]
/retitle ODC-7336: automation for customization-of-catalog-add-page-form feature file

**Pull request merged**